google/import_compute_instance_template_test.go

```diff
@@ -1,147 +0,0 @@
-package google
-
-import (
-	"testing"
-
-	"github.com/hashicorp/terraform/helper/acctest"
-	"github.com/hashicorp/terraform/helper/resource"
-)
-
-func TestAccComputeInstanceTemplate_importBasic(t *testing.T) {
-	t.Parallel()
-
-	resourceName := "google_compute_instance_template.foobar"
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckComputeInstanceTemplateDestroy,
-		Steps: []resource.TestStep{
-			resource.TestStep{
-				Config: testAccComputeInstanceTemplate_basic(),
-			},
-
-			resource.TestStep{
-				ResourceName:      resourceName,
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
-		},
-	})
-}
-
-func TestAccComputeInstanceTemplate_importIp(t *testing.T) {
-	t.Parallel()
-
-	resourceName := "google_compute_instance_template.foobar"
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckComputeInstanceTemplateDestroy,
-		Steps: []resource.TestStep{
-			resource.TestStep{
-				Config: testAccComputeInstanceTemplate_ip(),
-			},
-
-			resource.TestStep{
-				ResourceName:      resourceName,
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
-		},
-	})
-}
-
-func TestAccComputeInstanceTemplate_importDisks(t *testing.T) {
```

Comment:
any reason to leave an import step out of the equivalent for this test?

Reply:
A miss on my part. Added